backend/src/apiserver/resource/model_converter.go

```diff
@@ -253,28 +256,31 @@ func runtimeConfigToModelParameters(runtimeConfig *api.PipelineSpec_RuntimeConfi
 		return "", nil
 	}
 	var params []v1alpha1.Parameter
```

Comment:
the code is confusing to me. the runtime config is a v2 proto, but here we marshall the runtime config into a string through a v1alpha1 API proto. 

Can we start refactoring these code and separate out the v1 and v2 code path, making them hermetically avoiding referring to the proto in the other version?

Reply:
Yes, that's a good point! I will update the code accordingly.